docs/parameters.md

```diff
@@ -1,10 +1,53 @@
-# Using Parameters
+# Parameters
```

Comment:
This first line needs to be updated in `redskyops-www/build.sh` as well. You should also update the `/docs/metrics.md` to be consistent.

Reply:
Will do, thanks